### PR TITLE
move some inactive to emeritus_approvers, update oncall and build-adm…

### PR DESCRIPTION
**Pull request by BenTheElder**
…in lists

### CODE REVIEW

File: OWNERS
Comment (BenTheElder) [inline OWNERS:10]: TODO: if we're adding people here based on chair/lead status still, @alvaroaleman is missing.

File: OWNERS
Comment (BenTheElder) [inline OWNERS:10]: no longer at google => not in oncall. inactive on github.

File: OWNERS_ALIASES
Comment (BenTheElder) [inline OWNERS_ALIASES:13]: xref https://github.com/kubernetes/org/issues/3600, Juan is no longer in build admins oncall, nor am I since I left engprod.

File: config/jobs/kubernetes-sigs/kubetest2/OWNERS
Comment (BenTheElder) [inline config/jobs/kubernetes-sigs/kubetest2/OWNERS:9]: I stepped down from kubetest2, so I'm also stepping down from the CI config

File: config/jobs/kubernetes/repo-infra/OWNERS
Comment (BenTheElder) [inline config/jobs/kubernetes/repo-infra/OWNERS:2]: everyone in this list is inactive here for the past year+, @cpanato is not and this list was empty, so I'm voluntelling into reviewers on track for approver 🙃

File: gopherage/OWNERS
Comment (BenTheElder) [inline gopherage/OWNERS:10]: not focusing on this and other engprod tools I'm not actively interested in, too many things to do. this and tools below have other listed owners, plus repo root OWNERS.

**Comment by BenTheElder** [comment]
bah, fixed typo on @cpanato

File: config/jobs/kubernetes/repo-infra/OWNERS
Comment (BenTheElder) [inline config/jobs/kubernetes/repo-infra/OWNERS:2]: everyone in this list is inactive here for the past year+, @cpanato is not and this list was empty, so I'm voluntelling into reviewers on track for approver 🙃

**Comment by ameukam** [comment]
/lgtm
/hold

In case others want to comment.

**Review by cpanato — APPROVED** [review]
Thank you

/lgtm

**Comment by BenTheElder** [comment]
/hold cancel
I'm going to follow up with Alvaro in another PR tomorrow even though it looks like he's +1 :-)

**Pull request merged**
